cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -82,6 +82,42 @@ struct id_to_type_impl {
 template <cudf::type_id Id>
 using id_to_type = typename id_to_type_impl<Id>::type;
 
+/**
+ * @brief "Returns" the corresponding type that is stored on the device when using `cudf::column`
+ *
+ * Only the fixed_point::rep is stored on device when a `fixed_point` is stored in a `cudf::column`.
+ * For example, the `using decimal32 = fixed_point<int32_t, Radix::BASE_10>` means that `decimal32`
```

Comment:
I don't think you should quote details of the inner code in the API docs. Just say decimal32 is stored as int32.

Reply:
Done.